distribution/lib/Standard/Table/0.2.32-SNAPSHOT/src/Data/Table.enso

```diff
@@ -406,6 +407,32 @@ type Table
         new_columns = Table_Helpers.reorder_columns internal_columns=this.columns selector=columns position=position on_problems=on_problems warnings=warnings
         here.new new_columns
 
+    ## Returns a new table with the columns sorted by name according to the
+       specified sort method. By default, sorting will be according to
+       case-sensitive ascending order based on code points.
```

Comment:
If I remember correctly, in our design document we've been talking aobut grapheme clusters, not code points. Sorting by code points will work bad, especially for international cases.

Reply:
Fair. Well, what this option does is using the default `_.compare_to` on strings.
IMO changing the way we compare Texts is out of scope of this change (which just adds a new function on Tables), so I'd suggest modifying the default comparison method as part of the work on improving the Text API.
In that case that comment would stay as-is for now and be updated accordingly once that happens. Does that work?

Reply:
I have changed the docs to say that the order is according to `Text.compare_to`.

Will create a separate task to ensure that `Text.compare_to` is according to grapheme clusters.